src/polyhash/index.js

```diff
@@ -300,18 +384,22 @@ function _toTurfPolygon(inputPolygon) {
 
 
 /**
- * Convert the given polygon into a cluster of locationIds
+ * Convert a geometry into a cluster of locationIds
  *
- * @param {*} polygon, Array of coordinates or GeoJSON polygon
+ * @param {*} input, coordindates, locationIds, polygon or stringId
  * @param {*} locationIdPrecision
  */
-function toCluster(inputPolygon, locationIdPrecision) {
+function toCluster(input, locationIdPrecision) {
```

Comment:
Is there a reasonable default value we could assign to locationIdPrecision?
Then we could add this to a list of possible options to handle on clustering, e.g.
  minimiseLocationIds (default: true)
  breakdownEdges (default: false)

Names and default values can also be changed. What do you think @eamonnmcevoy ?

Reply:
Good idea, I would suggest 9 as a default precision since that is the cell precision we show on the map.

Reply:
Updated